auraed/src/cri/runtime_service.rs

```diff
@@ -115,72 +125,111 @@ impl runtime_service_server::RuntimeService for RuntimeService {
         // TODO Switch on "WASM" which is a field that we will add to the RunPodSandboxRequest
         // TODO We made the decision to create a "KernelSpec" *name structure that will be how we distinguish between VMs and Containers
 
-        // Initialize a new container builder with the AURAE_SELF_IDENTIFIER name as the "init" container running a recursive Auraed
-        let syscall = create_syscall();
-        let sandbox_builder = ContainerBuilder::new(
-            AURAE_SELF_IDENTIFIER.to_string(),
-            syscall.as_ref(),
-        );
-
-        // Spawn auraed here
-        // TODO Check if sandbox already exists?
-        let _spawned = spawn_auraed_oci_to(
-            Path::new(AURAE_BUNDLE_PATH)
-                .join(AURAE_SELF_IDENTIFIER)
-                .to_str()
-                .expect("recursive path"),
-            oci_builder.build().expect("building pod oci spec"),
-        );
-
-        let sandbox_id = metadata.name;
-        let mut sandbox = sandbox_builder
-            .with_root_path(Path::new(AURAE_PODS_PATH).join(sandbox_id.clone()))
-            .expect("Setting pods directory")
-            .as_init(Path::new(AURAE_BUNDLE_PATH).join(AURAE_SELF_IDENTIFIER))
-            .with_systemd(false)
-            .build()
-            .expect("failed building pod sandbox: ensure valid OCI spec and proper container starting point");
-
-        sandbox.start().expect("starting pod sandbox");
+        let mut sandbox: Option<Container>;
```

Comment:
It looks like you are using option  to initialize to none temporarily. The block can return the sandbox. 'let sandbox = { ... };'

Reply:
ack